src/GameSolution2.ts

```diff
@@ -0,0 +1,103 @@
+export class Game {
+    private _lastSymbol: string = ' ';
+    private _board: Board = new Board();
+
+    public Play(symbol: string, x: number, y: number) : void {
+        //if first move
+        if (this._lastSymbol == ' ') {
+            //if player is X
+            if (symbol == 'O') {
+                throw new Error("Invalid first player");
+            }
+        }
+        //if not first move but player repeated
+        else if (symbol == this._lastSymbol) {
+            throw new Error("Invalid next player");
+        }
+        //if not first move but play on an already played tile
+        else if (this._board.TileAt(x, y).Symbol != ' ') {
+            throw new Error("Invalid position");
+        }
+
+        // update game state
+        this._lastSymbol = symbol;
+        this._board.AddTileAt(symbol, x, y);
+    }
+
+    public Winner() : string {
+        //if the positions in first row are taken
+        if (this._board.TileAt(0, 0)!.Symbol != ' ' &&
+                this._board.TileAt(0, 1)!.Symbol != ' ' &&
+                this._board.TileAt(0, 2)!.Symbol != ' ') {
```

Comment:
this._board.TileAt(0, 2)!.Symbol != ' '  -> inaproiate intimacy + feature envy
this._board.TileAt(0, 2).isEmpty(). -> feature envy (yo are asking all time but you don't know the internals)
this._board.isWinner() -> no smells ( removes data class as well )